### ISSUE TITLE
Programmatically access and set page in pdf viewer

### ISSUE THREAD

**Issue by christopher-carmichael** [MicrosoftEdge/WebView2Feedback#3184]
**Is your feature request related to a problem? Please describe.**

I find it a gross oversight that we can not even get the page number of the current page in the pdf viewer. I can understand some of the controls, but my entire problem could be solved if I could get the value of the page number element. 

**Describe the solution you'd like and alternatives you've considered**
 
I would like more control of pdfs programmatically. I initially saw similar issues from years ago. The problem is there are almost no pdf viewer controls available for winui3 even third party so a first party solution would be nice.

**Comment by vickiez** [comment]
Hi @christopher-carmichael, I created an internal work item to track this feature request. We'll let you know when there is an update. In the meantime, can you share some more details about your scenario? Would #2712 meet your needs?

**Comment by christopher-carmichael** [comment]
No, #3178 could be useful and would be great. Issue #2712 is similar, however the user seems more interested in the position within the pdf instead of the actual index of the page.

My scenario is as such: I want to modify the pdf via other libraries and use webview2 pdf viewer to display the pdf. The pdf is then sent via api into document storage system. 

While the viewer allows things like rotate, it does not change the actual pdf.  I want to use the viewer as simply a viewer and do all modifications, deletes, inserts in code. To accomplish this I need to be able to get the current page within the pdf being viewed, or selected. 

My work around will be to only show one page at a time and have navigation buttons between pages, but this is not ideal for the user experience of my application.
